bfdd/bfdd_vty.c

```diff
@@ -142,8 +142,12 @@ static void _display_peer(struct vty *vty, struct bfd_session *bs)
 
 	vty_out(vty, "\t\tDiagnostics: %s\n", diag2str(bs->local_diag));
 	vty_out(vty, "\t\tRemote diagnostics: %s\n", diag2str(bs->remote_diag));
+	vty_out(vty, "\t\tPeer Type: %s\n",
+		BFD_CHECK_FLAG(bs->flags, BFD_SESS_FLAG_CONFIG) ? "configured" : "dynamic");
 
 	vty_out(vty, "\t\tLocal timers:\n");
+	vty_out(vty, "\t\t\tDetect-multiplier: %" PRIu32 "\n",
+		bs->detect_mult);
 	vty_out(vty, "\t\t\tReceive interval: %" PRIu32 "ms\n",
```

Comment:
not under Local timers, please.
I think better to add it at line 147 ( above)

Reply:
Since multiplier value is closely related to timers, i thought it would be logical to club them them under local and remote timers.

Reply:
we have local and remote multiplier value, like for local/remote diganostic.
why not having 2 specific lines for that outside of the timers.

Reply:
Unlike local/remote diagnostic, multiplier values are closely related to timers and determines the actual BFD timeout value. In my opinion, it is due to this close relation with timer values it makes more sense to club them together with timers instead of having 2 specific lines.

Reply:
I agree to keep it under timers, but please precise "multiplier" and "remote multiplier" keywords.

Reply:
Below is the sample output after the change, let me know how exactly you think we can modify to convey the right information.

BFD Peer:
        peer 12.12.3.1 interface Vlan101
                ID: 8
                Remote ID: 3
                Status: up
                Uptime: 58 second(s)
                Diagnostics: ok
                Remote diagnostics: ok
                Peer Type: dynamic
                Local timers:
                        Detect-multiplier: 3
                        Receive interval: 300ms
                        Transmission interval: 300ms
                        Echo transmission interval: 50ms
                Remote timers:
                        Detect-multiplier: 3
                        Receive interval: 100ms
                        Transmission interval: 100ms
                        Echo transmission interval: 50ms

Reply:
@pguibert6WIND : would appreciate your comment on this

Reply:
Hi Sumit, sorry for delay.
ok for this.
by the way, for json format, do you  have some keyword where to put Detect-multiplier and Remote-Detect Multiplier ?

Reply:
I have not modified for json
How about "detect-multiplier" and "remote-detect-multiplier", if we agree on this keywords i can add these changes

Reply:
Added "detect-multiplier", "remote-detect-multiplier", "peer-type" for json